posts/rust-wasm.md

```diff
@@ -1,30 +1,31 @@
 # Rust and the case for Web Assembly in 2018
-Published November th, 2017
+
+Published November 16th, 2017
 
 Every now and then a technology comes around that fundamentally changes the game
 and how things work. Java and the JVM created a world where the idea of portable
-code execution was possible, compile once run anywhere (as long as there was
+code execution was possible, "compile once run anywhere" (as long as there was
 a JVM available). JavaScript changed how the web worked and went from something
-to help manipulate how web pages looked based off user interaction, to it's
-modern incarnation of full applications built into the browser.
+to help manipulate how web pages looked based off user interaction, to its
+modern incarnation of full applications that run in the browser.
 
 ![xkcd comic number 1367 describing that web pages are apps now too][xkcd]
 
 I think we're at that next technology now and it's staring us in the face, but
-because it hasn't been used beyond hobbyist level interaction it's kind of hard
-to see why one might use it for their own purposes yet. I'm talking about Web
-Assembly (wasm for short). It's a big deal. We're talking an Internet unlike any
+because it hasn't been used beyond hobbyist level interaction, it's kind of hard
+to see why one might use it for their own purposes yet. I'm talking about
+WebAssembly ("wasm" for short). It's a big deal. We're talking an Internet unlike any
 that's been seen before and it's going to change everything.
 
 ## What is wasm?
 
 If you haven't heard of it wasm is a binary format to run programs at native
-speeds in the browser. What? That's crazy you might say, well here's an example
+speeds in the browser. What? "That's crazy" you might say, well here's an example
 of it [being used for a neural network to generate anime faces][moe_moe_kyun].
 The code for that isn't written in JavaScript. Now while that may not be your
 cup of tea you might be interested in the fact that someone put the
 [Godot Game Engine][godot] into the browser. This is all early days kind of stuff
-with wasm but it highlights an important thing, wasm is powerful, wasm is fast enough,
+with wasm but it highlights an important things: Wasm is powerful, wasm is fast enough,
```

Comment:
this one i actually disagree with, the name is just `wasm` and you don't need to capitalize things after a `:`.